diag_manager/fms_diag_file_object.F90

```diff
@@ -613,9 +642,11 @@ subroutine dump_file_obj(this, unit_num)
 end subroutine
 
 !< @brief Opens the diag_file if it is time to do so
-subroutine open_diag_file(this, time_step)
+subroutine open_diag_file(this, time_step, file_is_opened)
```

Comment:
They're called history files, not diag files.

Reply:
lol we don't call them history files anywhere in the code ...

Reply:
touche